schemas/2019-04-01/deploymentTemplate.json

```diff
@@ -63,6 +63,7 @@
                   { "$ref": "https://schema.management.azure.com/schemas/2016-06-01/Microsoft.RecoveryServices.json#/resourceDefinitions/vaults_extendedInformation" },
                   { "$ref": "https://schema.management.azure.com/schemas/2015-06-01/Microsoft.KeyVault.json#/resourceDefinitions/secrets" },
                   { "$ref": "https://schema.management.azure.com/schemas/2016-10-01/Microsoft.KeyVault.json#/resourceDefinitions/vaults_secrets" },
+                  { "$ref": "https://schema.management.azure.com/schemas/2016-10-01/Microsoft.KeyVault.json#/resourceDefinitions/vaults_accessPolicies" },
```

Comment:
Please could you also make this change to the other root schemas? 

* https://github.com/Azure/azure-resource-manager-schemas/blob/master/schemas/2014-04-01-preview/deploymentTemplate.json
* https://github.com/Azure/azure-resource-manager-schemas/blob/master/schemas/2015-01-01/deploymentTemplate.json